pyvo/registry/regtap.py

```diff
@@ -866,49 +879,45 @@ def service(self):
         See Also
         --------
         get_service : select a service of a specific service type.
-        list_services : return a list with all the available services.
+        list_interfaces : return a list with all the available interfaces to services.
         """
         if self._service is not None:
             return self._service
         self._service = self.get_service(service_type=None, lax=True)
         return self._service
 
-    def list_services(self, service_type: str = None):
-        """
-        List the services available for this registry record.
+    def list_interfaces(self, service_type: str = None):
+        """List the interfaces to services available for this registry record.
+
+        The interface objects in the list can then be used to instantiate the service with
+        the  `~pyvo.registry.regtap.Interface.to_service` method.
 
         Parameters
         ----------
         service_type : str, optional
-            Restricts the list to services corresponding to this service type.
+            Restricts the list to interfaces corresponding to services of this service type.
             The list of possible values can be printed with
             ``pyvo.registry.rtcons.SERVICE_TYPE_MAP.keys()``.
 
         Returns
         -------
         list
-            A list of service objects. They can be
-            `~pyvo.dal.scs.SCSService`, `~pyvo.dal.sia.SIAService`,
-            `~pyvo.dal.sia2.SIA2Service`, `~pyvo.dal.ssa.SSAService`,
-            `~pyvo.dal.sla.SLAService`, or `~pyvo.dal.tap.TAPService`.
+            A list of `~pyvo.registry.regtap.Interface` objects.
 
         See Also
         --------
-        get_service : when there is only one service of a specific service type.
+        get_service : when you already know that there is only one service of a specific service type.
 
         """
+        list_interfaces = [interface for interface in self.interfaces
```

Comment:
I can't say I'm a big fan of the list_ prefix here, because it sounds too much like a verb phrase (and hence a function).  I'd probably just call it ``interfaces`` or, if you think that's too close to ``interface``, interfaces_list.